documents/TestSuite/stdlib/color_management/color_management.mtlx

```diff
@@ -0,0 +1,623 @@
+<!--
+
+Basic image unit test with one image node for each variation in input type.
+
+-->
+<?xml version="1.0"?>
+<materialx version="1.36" cms="default_cms" colorspace="lin_rec709">
+   <nodegraph name="ng1">
+        <output name="image_output" type="surfaceshader" nodename="image_standard_surface" />
+        <standard_surface name="image_standard_surface" type="surfaceshader" xpos="6.29441" ypos="2.18094">
+          <input name="base" type="float" value="0.8" />
+          <input name="base_color" type="color3" value="1, 1, 1" nodename="image" />
+          <input name="diffuse_roughness" type="float" value="0"/>
```

Comment:
Minor: for all the inputs that just has default values we don't need to specify them here, since the nodedef will hold the default values anyway. It would make the tests more compact and easy to read if only the inputs with changed values were specifed. 

But I guess this is an export from BlackSky and _all_ the inputs get exported? So it's something we can fix on the BlackSky side in that case.